presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/AccumuloQueryRunner.java

```diff
@@ -171,58 +171,29 @@ public static Connector getAccumuloConnector()
         }
 
         try {
-            MiniAccumuloCluster accumulo = createMiniAccumuloCluster();
-            Instance instance = new ZooKeeperInstance(accumulo.getInstanceName(), accumulo.getZooKeepers());
-            connector = instance.getConnector(MAC_USER, new PasswordToken(MAC_PASSWORD));
-            LOG.info("Connection to MAC instance %s at %s established, user %s password %s", accumulo.getInstanceName(), accumulo.getZooKeepers(), MAC_USER, MAC_PASSWORD);
+            FixedHostPortGenericContainer<?> accumuloContainer = new FixedHostPortGenericContainer<>("prestodev/accumulo:latest");
+            accumuloContainer.withFixedExposedPort(ACCUMULO_MASTER_PORT, ACCUMULO_MASTER_PORT);
+            accumuloContainer.withFixedExposedPort(ACCUMULO_TSERVER_PORT, ACCUMULO_TSERVER_PORT);
+            accumuloContainer.withFixedExposedPort(ZOOKEEPER_PORT, ZOOKEEPER_PORT);
```

Comment:
do we need to use fixed ports? It would be nice to have them random.

Reply:
ZooKeeper doesn't need to be fixed -- I'll change that to random. But Accumulo needs to be fixed.  Clients discover the services from ZooKeeper and the `ip:port` that is advertised is the internal port that they are bound to.